src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -198,11 +198,19 @@ class AbstractIntegrationDetailedView<
     return {organization, features};
   }
 
+  cleanTags(featureData) {
+    const tags = featureData.map(feature =>
+      feature.featureGate.replace(/integrations/g, '').replace(/-/g, ' ')
+    );
+    return [...new Set(tags)];
```

Comment:
@ceorourke You could also use Lodash's uniq. See: https://lodash.com/docs/4.17.15#uniq